IM21DoorStatusEnum.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const IM21_DOOR_STATE = {
+    OPEN: '02',
+    CLOSED: '00',
+    HEARTBEAT_OPEN: '0A',
+    HEARTBEAT_CLOSED: '08',
+    LOW_BATT: '04'
+};
```

Comment:
We are moving away from using semi-colons. Could you please take them out of any new lines of code that you write? That will make it easier when we make the linter more strict (which may be coming soon).